src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -368,11 +370,20 @@ const std::vector<impl_desc_type>& Convolution::getDefaultImplPriority() {
         impl_desc_type::ref,
     };
 
+    priorities.erase(std::remove_if(priorities.begin(),
+                                    priorities.end(),
+                                    [](impl_desc_type type) {
+                                        return !isBrgConvAvailable() && (type == impl_desc_type::brgconv_avx2_1x1 ||
```

Comment:
Maybe it is reasonable to delete all brgconv impls in that case. Just seems much more logical.
Like `!isBrgConvAvailable() && (type & impl_desc_type::brgconv)`

Reply:
Seems more reasonable to remove all the brgconv impls as @dmitry-gorokhov suggested.

Reply:
changed to delete all brgconv impls